### ISSUE TITLE
BEiTv2 MIM accuracy 

### ISSUE THREAD

**Issue by ckddls1321** [microsoft/unilm#885]
**Describe**
Model I am using BEiTv2 ViT-L/16.

I pre-trained 1K using vqkd tokenizer, it seems MIM(Masked Image Model) accuracy does not go up to 40~50%.

Can you provide log or accuracy reference of pre-training on 1K with tokenizer? 
Also if you have evaluation results on 1K, can you share us?

**Comment by ckddls1321** [comment]
Thank you. We also have similar results in pre-training stage. I was wondering about the right MIM accuracy when model learns VQKD codebook. 

BTW, when we evaluate 1600 epoch checkpoint at https://conversationhub.blob.core.windows.net/beit-share-public/beitv2/beitv2_large_patch16_224_pt1k.pth, it does not show right accuracy when we use teacher https://conversationhub.blob.core.windows.net/beit-share-public/beitv2/vqkd_encoder_base_decoder_3x768x12_clip-d5036aa7.pth. However, 300 epoch checkpoint shows similar results as trained. 
If you don't mind, can you confirm 1600 epoch checkpoint and teacher once again?

**Comment by ckddls1321** [comment]
> Hello,
> 
> The MIM accuracy is about 16% when using vqkd tokenzier to pretrain ViT-L/16. ![image](https://user-images.githubusercontent.com/26346329/194298381-e617d56c-53db-4e82-a9d7-309f4b8e9f31.png)
> 
> When the pretraining schedule is 1600, the accuracy increases slightly: ![image](https://user-images.githubusercontent.com/26346329/194299080-4178bdff-f698-4ff3-b341-f4218ddcda42.png)

Thank you. We also have similar results in pre-training stage. I was wondering about the right MIM accuracy when model learns VQKD codebook.

BTW, when we evaluate 1600 epoch checkpoint at https://conversationhub.blob.core.windows.net/beit-share-public/beitv2/beitv2_large_patch16_224_pt1k.pth, it does not show right accuracy when we use teacher https://conversationhub.blob.core.windows.net/beit-share-public/beitv2/vqkd_encoder_base_decoder_3x768x12_clip-d5036aa7.pth. However, 300 epoch checkpoint shows similar results as it trained.
If you don't mind, can you confirm 1600 epoch checkpoint and teacher once again?

**Comment by ckddls1321** [comment]
Thank you! We also have the same trends during training. 

During training, Models perform better on every scale to predict masked patches based on visual information.
Even it would be hard to predict all scales correctly. The lastest checkpoint performs better.

**Comment by ckddls1321** [comment]
Thank you! We also have the same trends during training. 

During training, Models perform better on every scale to predict masked patches based on visual information.
Even it would be hard to predict all scales correctly. The lastest checkpoint performs better.

**Comment by zengshao0622** [comment]
What about the BEiTv2 ViT- B/16？
